sdk/search/Azure.Search.Documents/tests/Samples/Readme.cs

```diff
@@ -174,6 +178,56 @@ public async Task Options()
             #endregion Snippet:Azure_Search_Tests_Samples_Readme_Options
         }
 
+        [Test]
+        [SyncOnly]
+        public async Task CreateIndex()
+        {
+            await using SearchResources resources = await SearchResources.CreateWithNoIndexesAsync(this);
+            Environment.SetEnvironmentVariable("SEARCH_ENDPOINT", resources.Endpoint.ToString());
+            Environment.SetEnvironmentVariable("SEARCH_API_KEY", resources.PrimaryApiKey);
+
+            #region Snippet:Azure_Search_Tests_Samples_Readme_CreateIndex
+            Uri endpoint = new Uri(Environment.GetEnvironmentVariable("SEARCH_ENDPOINT"));
+            string key = Environment.GetEnvironmentVariable("SEARCH_API_KEY");
+
+            // Create a service client
+            AzureKeyCredential credential = new AzureKeyCredential(key);
+            SearchServiceClient client = new SearchServiceClient(endpoint, credential);
+            /*@@*/ client = resources.GetServiceClient();
+
+            // Create the index
+            //@@SearchIndex index = new SearchIndex("hotels")
+            /*@@*/ SearchIndex index = new SearchIndex(Recording.Random.GetName())
```

Comment:
It would probably be safer to structure this as 

```C#
SearchIndex index = new SearchIndex("hotels") /*@@*/;
/*@@*/ index = new SearchIndex(Recording.Random.GetName())
```

so we're still compiling the sample code and will refactor with tooling.  We can do that in Preview 4 though.